XenAdmin/Commands/Controls/PoolRemoveServerToolStripMenuItem.cs

```diff
@@ -0,0 +1,106 @@
+﻿/* Copyright (c) Citrix Systems, Inc. 

```

Comment:
We can discuss if we want to take this change, but generally I think this is what the user would expect to see on the pool menu. I think the pevious item was confusing because if the user selected a server, I doubt they would go to the pool menu to remove it.

Reply:
I like these changes. Another inconsistency is that the context menu for the host doesn't show the "Remove from Pool" item for master, but the main menu shows it and allows it.